controllers/list/users/SelectReviewerListHandler.inc.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * @file controllers/list/users/SelectReviewerListHandler.inc.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SelectReviewerListHandler
+ * @ingroup classes_controllers_list
```

Comment:
correct?

Reply:
I think so? I've put all of the new list handlers into `classes_controllers_list`, but I'm not sure if I'm doing that docblock right.